x/authz/simulation/operations.go

```diff
@@ -20,10 +20,11 @@ import (
 )
 
 // authz message types
-const (
-	TypeMsgGrantAuthorization  = "/cosmos.authz.v1beta1.Msg/GrantAuthorization"
-	TypeMsgRevokeAuthorization = "/cosmos.authz.v1beta1.Msg/RevokeAuthorization"
-	TypeMsgExecDelegated       = "/cosmos.authz.v1beta1.Msg/ExecAuthorized"
+var (
+	// FIXME Remove `Request` suffix
+	TypeMsgGrantAuthorization  = sdk.MsgRoute(&types.MsgGrantAuthorizationRequest{})
+	TypeMsgRevokeAuthorization = sdk.MsgRoute(&types.MsgRevokeAuthorizationRequest{})
+	TypeMsgExecDelegated       = sdk.MsgRoute(&types.MsgExecAuthorizedRequest{})
```

Comment:
@robert-zaremba Is removing the `Request` suffix part of #9114?

Reply:
I think it is